com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -178,7 +179,7 @@ public Color clearColor
         internal bool overrideCameraTarget { get; set; }
         internal bool isBlitRenderPass { get; set; }
 
-        internal bool useNativeRenderPass { get; set; }
+        public bool useNativeRenderPass { get; set; }
```

Comment:
As this is public it needs at least some api documentation

Reply:
Actually we should not make this public. Why is that? @jonuuukas ?

Reply:
initially this was needed to disable the API for injected passes, but as we did change how that works this, i think, is no longer necessary. Let me quickly test this and if all is good, i'll revert it